Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml

```diff
@@ -690,15 +690,19 @@
     sprite: Objects/Weapons/Guns/Projectiles/projectiles2.rsi
     layers:
       - state: grenade
+  - type: StaminaDamageOnCollide
```

Comment:
please comment on your changes in this file something like # imp so in the case that upstream alters baton grenades then whoever does the merge knows that we fucked with the file